code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -119,6 +119,7 @@
 				atoms = orange(3, target)
 			for(var/atom/movable/A in atoms)
 				if(A.anchored) continue
```

Comment:
You can just do if(A.anchored || A.move_resist == INFINITY)

instead of two separate line. They're mostly the same thing after all.

Reply:
That is true, changing